element_array_ephys/export/nwb/requirements.txt

```diff
@@ -0,0 +1,4 @@
+dandi
+git+https://github.com/catalystneuro/nwb-conversion-tools.git
+git+https://github.com/SpikeInterface/spikeinterface.git
+git+https://github.com/bendichter/python-neo.git@2d0ba5213f6e2add29dd13b2131650e3325edb23
```

Comment:
Why [this particular python-neo commit](https://github.com/bendichter/python-neo/tree/2d0ba5213f6e2add29dd13b2131650e3325edb23)? It would help me to have that documented so that we can more readily review python-neo updates down the road, assess whether or not to include or fork for ourselves, etc.

Reply:
I was having import errors on the latest that the NEO team has not been able to replicate, but this state imported without error. I'll dig a little deeper to see if I can figure out the root cause.